frontend/components/FleetsDropdown/_styles.scss

```
@@ -0,0 +1,142 @@
+  &__menu-list:focus,
+  &__menu-list:focus-visible {
+    outline: none;
+  }
+
+  // Scroll-fade anchor sits at the bottom of the options area with
+  // zero flow height, so toggling the fade's visibility never changes
+  // MenuList.scrollHeight (which would clamp scrollTop and jump at the
+  // very bottom of the list). Negative bottom pulls the anchor past
+  // MenuList's pad-small paddingBottom so the gradient sits flush
+  // against the outer edge of the scroll area rather than 8px above it.
+  //
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)